etc/kayobe/kolla-image-tags.yml

```diff
@@ -44,5 +44,5 @@ kolla_image_tags:
   ovn:
     rocky-9: 2023.1-rocky-9-20240809T102431
   rabbitmq:
-    rocky-9: 2023.1-rocky-9-20240823T101942
+    rocky-9: 2023.1-rocky-9-20240925T133913
```

Comment:
New tags need a release note, and we should probably be quite careful when promoting new container images for RabbitMQ. You could try kicking off a Multinode using CI with this branch to test?

Reply:
versions didn't change for main antelope version 3.11, and 3.12. only 3.12 changed:

```
new repo:
rabbitmq-server-3.11.28-1.el8.noarch.rpm
rabbitmq-server-3.12.14-1.el8.noarch.rpm
rabbitmq-server-3.13.7-1.el8.noarch.rpm

old repo:
rabbitmq-server-3.11.28-1.el8.noarch.rpm
rabbitmq-server-3.12.14-1.el8.noarch.rpm
rabbitmq-server-3.13.4-1.el8.noarch.rpm
```